packages/webr/R/eval.R

```diff
@@ -15,26 +15,23 @@
 #' @param handlers If `TRUE`, execute the R code using a [tryCatch], with
 #' handlers in place.
 #'
+#' @useDynLib webr
 evalRCode <- function(code, conditions = TRUE, streams = FALSE,
   autoprint = FALSE, handlers = TRUE
 ) {
   res <- NULL
-  errors <- NULL
-  warnings <- NULL
-  messages <- NULL
-  stderr <- NULL
-  stdout <- NULL
+  out <- .Call("new_output_connections")
```

Comment:
Needs a comment that we can't use individual `textConnection()` for stdout and stderr because we need to multiplex them in a single stream.

Reply:
Prefix C symbols with `ffi_`

```suggestion
  out <- .Call("ffi_new_output_connections")
```

Reply:
By the way this is doing a symbol search across all dlopened DLLs. Should at least pass the package as argument to avoid this search and prevent unexpected masking. But as argued above, it's better to use symbol registration so that we can `.Call()` into a C function pointer directly.